net/libfab/libfab.c

```diff
@@ -764,6 +764,14 @@ static void libfab_poller(struct m0_fab__tm *tm)
 	libfab_tm_event_post(tm, M0_NET_TM_STARTED);
 	while (tm->ftm_state != FAB_TM_SHUTDOWN) {
 		do {
+			while(M0_FI_ENABLED("fail-trywait")) {
```

Comment:
Use ```do { ... } while (M0_FI_ENABLED("fail-trywait"))``` instead to avoid fi_trywait() call duplication.

Reply:
Done.